src/source/Ice/TurnConnection.c

```diff
@@ -713,7 +713,9 @@ STATUS turnConnectionSendData(PTurnConnection pTurnConnection, PBYTE pBuf, UINT3
 
     if (STATUS_FAILED(retStatus)) {
         DLOGW("iceUtilsSendData failed with 0x%08x", retStatus);
-        retStatus = STATUS_SUCCESS;
+        if (retStatus != STATUS_SOCKET_CONNECTION_CLOSED_ALREADY) {
```

Comment:
There might be a class of returned status values that we might chose to ignore and not only this one error. Suggest adding a macro that can be expanded